tests/integration-test/src/tests/basic.rs

```diff
@@ -123,187 +114,126 @@ fn test_load_unload_tc() {
 
     let mut uuids = vec![];
     let mut rng = rand::thread_rng();
+
     // Install a few tc programs
-    for _ in 0..10 {
-        let priority = rng.gen_range(1..255);
-        let uuid = add_tc_pass(
-            "ingress",
-            DEFAULT_BPFD_IFACE,
-            priority,
-            Some(globals.clone()),
-            Some(proceed_on.clone()),
-        );
-        uuids.push(uuid.unwrap());
+    for lt in LOAD_TYPES {
+        for _ in 0..5 {
+            let priority = rng.gen_range(1..255);
```

Comment:
I think @dave-tucker implemented this one.  I assume it was because he put the tc program adds in a loop and it was easier to generate priorities that were most likely different this way.